jni/Android.mk

```diff
@@ -47,7 +47,7 @@ INIPARSER_INCLUDES := $(INIPARSER_ROOT)/src
 
 LOCAL_ARM_NEON      := true
 LOCAL_CFLAGS        += -pthread -fopenmp -fexceptions
-LOCAL_CXXFLAGS      += -std=c++11 -frtti -fexceptions
+LOCAL_CXXFLAGS      += -std=c++14 -frtti -fexceptions
```

Comment:
So, nntrainer supports c++ version is greater or equal than 14. I think it must be noticed through README.md. Please fix it.